uwrt_mars_rover_control/src/uwrt_mars_rover_control_node.cpp

```diff
@@ -1,16 +1,97 @@
+#include <ros/console.h>
+#include <sys/mman.h>
+
 #include <cstdlib>
 
 #include "uwrt_mars_rover_control/uwrt_mars_rover_hw_control_loop_real.h"
 
-int main(int argc, char **argv) {
-  ros::init(argc, argv, "uwrt_mars_rover_control_node");
+bool hasRealtimeKernel() {
```

Comment:
can we put a few comments in this code

Reply:
i added some docstrings and some comments. is it clear enough?